salt/_modules/metalk8s_blkid_c.py

```diff
@@ -0,0 +1,138 @@
+# coding: utf-8
+"""Thin wrapper exposing libblkid functions."""
+
+import ctypes
+import ctypes.util
+
+soname = ctypes.util.find_library('blkid')
+if soname is None:
+    raise Exception('cannot find blkid library')
```

Comment:
Would an `ImportError` make sense?

Reply:
Yup, I think it's better to use `ImportError` here.
Done.